aw-server/src/config.rs

```diff
@@ -48,22 +48,22 @@ fn default_address() -> String {
 }
 
 fn default_port() -> u16 {
-    match Environment::active().expect("Failed to get current environment") {
-        Environment::Production => 5600,
-        Environment::Development => 5666,
-        Environment::Staging => panic!("Staging environment not supported"),
+    if is_production() {
+        5600
+    } else {
+        5666
     }
 }
 
 fn default_cors() -> Vec<String> {
-    match Environment::active().expect("Failed to get current environment") {
-        Environment::Production => Vec::<String>::new(),
-        Environment::Development => Vec::<String>::new(),
-        Environment::Staging => panic!("Staging environment not supported"),
-    }
+    Vec::<String>::new()
+}
+
+pub fn is_production() -> bool {
+    !is_testing()
 }
 
-fn is_testing() -> bool {
+pub fn is_testing() -> bool {
```

Comment:
This should not be public, the info should be shared through the config instead.
This can't be a function without a state once we add the "--testing" option, let's get ownership right already.

Reply:
This was supposed to be a small PR, and what you're asking for sounds like a lot of work that I'm not sure how I'd write, so merge and fix that later?

Reply:
Not too much work if you know how to do it, I can fix it later. This is till better than before!